### ISSUE TITLE
Add a way to force push from the Graph

### ISSUE THREAD

**Issue by sergeibbb** [gitkraken/vscode-gitlens#3493]
The graph doesn't have "Push" button that is presented in other Git UI products:
![image](https://github.com/user-attachments/assets/8c34baea-b6f2-4f16-9ea7-a6ff0f8062b5)

Here what we have in GitLens Graph:

![image](https://github.com/user-attachments/assets/56836f2a-d464-4386-855d-47103db8d925)

User needs to Pull first. After you pull it will be a push button.

But in some cases user doesn't want to pull. For example, it's after a rebase and a force push is needed.

To sum up, an ability to perform a force push from the Graph is desired.

**Comment by d13** [comment]
Thinking in this scenario, we make the pull a split-dropdown button and offer the force push in the dropdown.
